packages/web/docs/src/app/layout.tsx

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { ReactNode } from 'react';
+import localFont from 'next/font/local';
+import { Layout, useConfig } from 'nextra-theme-docs';
+import { Head } from 'nextra/components';
+import { getPageMap } from 'nextra/page-map';
```

Comment:
all dependencies should be imported from `@theguild/components`

Reply:
Okay so this goes down to the `@theguild/components/server` import.